addons/web/static/src/legacy/js/libs/bootstrap.js

```diff
@@ -148,3 +148,7 @@ ScrollSpy.prototype._activate = function (target) {
     }
     bootstrapSpyActivateFunction.apply(this, arguments);
 };
+
+Dropdown.prototype._detectNavbar = function () {
+    return false;
+};
```

Comment:
> Je me demande s'il ne faudrait pas overrider _detectNavbar  uniquement dans Website et limiter la désactivation à header .navbar plutot que .navbar pour ne rien changer par rapport aux versions précédentes.. Qu'en penses tu ?
Je pense que BS n'active pas le popper sur les dropdowns dans les navbars pour des raisons de performance, il considere peut-etre que dans une navbar on en a pas besoin car on devrait savoir où se place chaques éléments.

@bvr-odoo Replying here so there is confirmation on the related diff. I think it is ok this way but you r+ what you prefer :+1: However, in any case, this **really needs** comments in the code please :wink: